cds_migrator_kit/circulation/users/cli.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2015-2018 CERN.
+#
+# cds-migrator-kit is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""CDS Migrator Circulation Items CLI."""
+import glob
+import json
+import logging
+import os
+
+import click
+from flask import current_app
+
+from invenio_accounts.models import User
+from invenio_db import db
+from invenio_userprofiles.models import UserProfile
+from invenio_oauthclient.models import UserIdentity, RemoteAccount
```

Comment:
If you try this in the standalone installation of cds-migrator-kit, it will fail because there is no invenio-oauthclient.
We want this right? I also think it is probably better, but just to be sure.

Reply:
I can try move the imports inside the command but probably we want to fail yes.

Reply:
I moved all the imports inside the users method as the tests were failing because of the imports.